crypto/cipher/aes_gcm_wssl.c

```diff
@@ -405,7 +386,11 @@ static srtp_err_status_t srtp_aes_gcm_wolfssl_decrypt(void *cv,
     int err;
 
     if (c->dir != srtp_direction_encrypt && c->dir != srtp_direction_decrypt) {
-        return (srtp_err_status_bad_param);
+        return srtp_err_status_bad_param;
+    }
+
+    if (*dst_len < src_len - c->tag_len) {
```

Comment:
Is the subtraction correct?

Reply:
This is a decrypt function so the tag len will be removed from the src, so the dst_len must be greater then or equal to the src_len minus the tag_len .
There should probably be a pre check that src_len >= tag_len, will add that

Reply:
Yeah, that made sense this morning as I looked at it.